src/dwi/tractography/connectome/exemplar.h

```diff
@@ -35,12 +35,13 @@ class Exemplar : private Tractography::Streamline<float>
 { MEMALIGN(Exemplar)
   public:
     using Tractography::Streamline<float>::point_type;
-    Exemplar (const size_t length, const NodePair& nodes, const std::pair<point_type, point_type>& COMs) :
+    Exemplar (const size_t exemplar_index, const size_t length, const NodePair& nodes, const std::pair<point_type, point_type>& COMs) :
         Tractography::Streamline<float> (length, { 0.0f, 0.0f, 0.0f }),
         nodes (nodes),
         node_COMs (COMs),
         is_finalized (false)
     {
+      index = exemplar_index;
```

Comment:
Why is this needed? It doesn't look like it's used anywhere else...

Reply:
Not needed; just distracting when you're trawling through `gdb` outputs trying to find the source of an issue, and you keep seeing streamlines that claim to have an index of one quadrillion or whatever.